apps/antalmanac/src/components/RightPane/SectionTable/CourseInfoBar.tsx

```diff
@@ -190,8 +192,11 @@ const CourseInfoBar = (props: CourseInfoBarProps) => {
                     const currentTarget = event.currentTarget;
                     void togglePopover(currentTarget);
                 }}
+                style={{ minWidth: 200 }}
             >
-                {`${deptCode} ${courseNumber} | ${courseTitle}`}
+                <span
+                    style={{ whiteSpace: 'nowrap', overflow: 'hidden', textAlign: 'left' }}
```

Comment:
```suggestion
                    style={{ whiteSpace: 'nowrap', overflow: 'hidden', textOverflow: 'ellipsis', }}
```
No need to align text left, but we do want there to be visual indicator of overflow, which we can do with `textOverflow: 'ellipsis'`